src/lib/messenger.js

```diff
@@ -23,6 +23,19 @@ export default function messenger(bot) {
       const text = message.text.toLowerCase();
       const user = message.user || {};
 
+      function stripText(input) {
+        let commandText = '';
+        let command = '';
+
+        if (input.indexOf(' ') !== -1 && input.startsWith('/')) {
```

Comment:
I think I see what you mean, should we just add it to the if statement that we're about to extract into a function?
Or a separate test?

Not debating now, just genuinely curious, but is it even possible for it to be undefined?
Or is it just proper programming to always include error catching however minute the possibilities are?